deploy/crds/toolchain.openshift.dev_tektoninstallations_crd.yaml

```diff
@@ -33,6 +33,35 @@ spec:
           type: object
         status:
           description: TektonInstallationStatus defines the observed state of TektonInstallation
+          properties:
+            conditions:
+              description: 'Conditions is an array of current TektonInstallation conditions
+                Supported condition types: TektonInstalled, FailedToInstallTekton'
```

Comment:
I guess, after change of condition type, we need to run generate to correct this too.

Reply:
addressed in ba76ab5